lookup-table.yaml

```diff
@@ -212,3 +212,15 @@
     - utility
     - library
     - api
+- id: lambdabettergrass
```

Comment:
Please move this entry. The lookup table should be sorted alphabetically:
First, alphabetically by the publisher's name. Then, alphabetically by the id.